Automattic/jetpack
# Update the logo package to output SVG shapes

NOTE: don't merge. Will be merged after the packages feature branch is merged.

This PR adds a gray version of the Jetpack logo to the logo package.

This is a blocker for the VaultPress UI restyle in https://github.com/Automattic/vaultpress/pull/6

<img width="286" alt="Captura de Pantalla 2019-06-03 a la(s) 18 44 29" src="https://user-images.githubusercontent.com/1041600/58836638-a61cc900-862f-11e9-95c7-fa8f1e62b7ba.png">

#### Changes proposed in this Pull Request:

* remove the URL argument in constructor
* remove the SVG file
* update the render method to return an SVG string. Having these SVG shapes allows us to stylize them using CSS
* update unit tests
* update style for the only usage of this package in Jetpack, in the connection banner

#### Testing instructions:






Check the Jetpack connection banner 
* verify they're SVG shapes
* ensure it's displayed correctly
* run unit tests and ensure they pass

#### Proposed changelog entry for your changes:

* Not needed. Internal use.
